lego.sh

```diff
@@ -15,12 +15,25 @@ if [ "${BASEDIR}" != "${SSL_PATH}" ]; then
 	exit 1
 fi
 
+# Added support for SAN
+CERT_NAME=""
+HOSTS_ARGS=""
+for DOM in $(echo $CERT_HOST | tr "," "\n")
+do
+	if [ -z "$CERT_NAME" ]
+	then
+		CERT_NAME $DOM
+	fi
+    
+	HOSTS_ARGS+="-d $DOM"
```

Comment:
Did this work for you? Ash or busybox or whatever shell this thing is running doesn't seem to like `+=` for concatenation:

````
# sh -x test.sh
+ FOO=-d foo
+ FOO+=-d bar
test.sh: line 1: FOO+=-d bar: not found
+ echo -d foo
-d foo
```

Reply:
I used a test script on mac, let my check over the udm...

Reply:
Should be replaced with:
`HOST_ARGS="${HOST_ARGS} -d ${DOM}`